techniques/systemSettings/misc/rudderSNMP/1.0/metadata.xml

```diff
@@ -0,0 +1,22 @@
+<TECHNIQUE name="Rudder SNMP Configuration">
+  <DESCRIPTION>Configuration of the SNMP service and rudder agent monitoring</DESCRIPTION>
```

Comment:
You need to elaborate a bit more.
Snmpd is installed if it is not already there
The configuration is left untouched except for an extension -> the user can do anything, his change w'ont be overrided.